src/titleEscrow.ts

```diff
@@ -0,0 +1,31 @@
+import {ethers} from "ethers";
+import {EthereumAddress} from "./types";
+import {getWallet} from "./provider";
+import {abi as TitleEscrowABI, bytecode as TitleEscrowBytecode} from "../build/contracts/TitleEscrow.json";
+
+export class TitleEscrow {
+  wallet: ethers.Wallet;
+
+  constructor({wallet = getWallet()}: {wallet: ethers.Wallet | undefined}) {
+    if (!wallet) {
+      throw new Error("TitleEscrow requires a wallet to be supplied at initialization");
+    }
+    this.wallet = wallet;
+  }
+
+  async deployEscrowContract(
```

Comment:
can put this as a static method under the class https://github.com/Open-Attestation/oa-token/blob/a412863b6c94804bb4a681ffabf6392bc828ddc9/src/owner.ts#L80

and take a wallet as a param with getWallet() default